src/api/python/cvc4.pxi

```diff
@@ -1503,6 +1504,98 @@ cdef class Term:
         term.cterm = self.cterm.iteTerm(then_t.cterm, else_t.cterm)
         return term
 
+    def toPythonObj(self):
+        '''
+        Converts a constant value Term to a Python object.
+        Requires isConst to hold.
+
+        Currently supports:
+          Boolean -- returns a Python bool
+          Int     -- returns a Python int
+          Real    -- returns a Python Fraction
+          BV      -- returns a Python int (treats BV as unsigned)
+          Array   -- returns a Python dict
+                  -- the constant array base is the value for the string '*'
+        '''
+
+        if not self.isConst():
+            raise RuntimeError("Cannot call toPythonObj on a non-const Term")
+
+        string_repr = self.cterm.toString().decode()
+        assert string_repr
+        sort = self.getSort()
+        res = None
+        if sort.isBoolean():
+            if string_repr == "true":
+                res = True
+            else:
+                assert string_repr == "false"
+                res = False
+        elif sort.isInteger():
+            is_neg = False
+            updated_string_repr = string_repr
```

Comment:
I think this could be simplified: `updated_string_repr = string_repr.strip('()') .replace(' ', '')`and `res = int(updated_string_repr)`. You shouldn't need the additional sign check.

Reply:
Good point -- addressed in 3ba57ed7b00f65e1c1263934a07a90f62e67e645